astra/src/test/java/com/slack/astra/zipkinApi/ZipkinServiceSpanConversionTest.java

```diff
@@ -36,7 +38,7 @@ private static LogWireMessage makeWireMessageForSpans(
   private static List<LogWireMessage> generateLogWireMessagesForOneTrace(
       Instant time, int count, String traceId) {
     List<LogWireMessage> messages = new ArrayList<>();
-    for (int i = 1; i <= count; i++) {
+    for (long i = 1; i <= count; i++) {
```

Comment:
So, before the int in the for loop was coerced to long by the call to `makeWireMessageForSpans`, but when I changed the type signature to Object, that coercion wouldn't happen anymore. This change ensures that the duration is a long. Maybe it would be clearer to convert i to a long where `makeWireMessageForSpans` is called though

Reply:
Oh, If I dropped the long change here entirely, and instead just explicitly check both cases in the test I introduced, I think that would be better.

Reply:
I did the above and reformatted the changeset to make the linter happy.